tools/azure_dependencies.sh

```diff
@@ -17,6 +17,7 @@ elif [ "${TEST_MODE}" == "pip-pre" ]; then
 	python -m pip install --progress-bar off --upgrade --only-binary ":all" "vtk<=9.0.1"
 	python -m pip install --progress-bar off https://github.com/pyvista/pyvista/zipball/main
 	python -m pip install --progress-bar off https://github.com/pyvista/pyvistaqt/zipball/main
+	python -m pip install --progress-bar off pooch
```

Comment:
This is not needed because the last line includes `pip install -r requirements_testing.txt` and it's in there